packages/get-platform/src/binaryTargets.ts

```diff
@@ -26,6 +26,7 @@ export type BinaryTarget =
   | 'freebsd12'
   | 'freebsd13'
   | 'freebsd14'
+  | 'freebsd15'
```

Comment:
You are also missing it in the `binaryTargets` array below

Reply:
yes, my bad, it was unstaged ;)